.ci/install.sh

```diff
@@ -29,6 +30,7 @@ python3 -m pip install -U pytest
 python3 -m pip install -U pytest-cov
 python3 -m pip install pyroma
 python3 -m pip install test-image-results
+python3 -m pip install meson
```

Comment:
Does meson need to be added to the requirements.txt?

Reply:
I don't think so. It's a build dependency of dav1d so it's used when building libavif in CI, but it's not otherwise required to run tests.

Reply:
This line is no longer present.